.vale/styles/RedHat/ObviousTerms.yml

```
@@ -0,0 +1,16 @@
+---
+extends: existence
+level: suggestion
+# link:
+message: "Consider not documenting %s because it is self-explanatory."
+scope: raw
+# source:
+action:
+  name: remove
+tokens:
+  - '\*User\*'
```

```suggestion
  - '\*?User\*? field'
```
Is `**User**` also a valid bold formatting in AsciiDoc?

---

Also can we add "field" as extra text in these rules to make it cast iron in terms on only highlighting these fields and not some other valid usage of these terms.